codespell_lib/data/dictionary.txt

```diff
@@ -25132,10 +25386,12 @@ watn->want
 wavelenght->wavelength
 wavelenghts->wavelengths
 wavelnes->wavelines
+wayoints->waypoints
 wayword->wayward
 weant->want, wean,
 weaponary->weaponry
 weas->was
+webaserver->web server
```

Comment:
IMO `webserver` should also be valid.

Reply:
Added webaserver->web server, webserver,